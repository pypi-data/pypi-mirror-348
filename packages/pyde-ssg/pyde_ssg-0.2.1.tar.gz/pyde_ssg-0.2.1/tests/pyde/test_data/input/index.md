---
title: Website
layout: index
---
This is the main page of the website!
