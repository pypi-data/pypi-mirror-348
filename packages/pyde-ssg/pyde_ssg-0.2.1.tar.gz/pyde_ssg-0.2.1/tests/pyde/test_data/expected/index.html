<!DOCTYPE html>
<html lang="en-US">
	<head>
		<title>Website</title>
		<link rel="stylesheet" href="/styles/page.css">
		<script src="/js/script.js" type="module"></script>
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<meta charset="UTF-8">
	</head>
	<body>
		<header id="site-header">
			<h1><a href="https://www.example.com">Website</a></h1>
		</header>
		<main>
			<p>This is the main page of the website!</p>
			<div class="post">
				<div class="post-title">Another Post</div>
				<div class="word-count">Word count: 6</div>
				<div class="excerpt">
					<p>A short post with some tags.</p>
				</div>
			</div>
			<div class="post">
				<div class="post-title">Some Post</div>
				<div class="word-count">Word count: 62</div>
				<div class="excerpt">
					<p>This is a post wherein markdown is demonstrated. Watch how I
					choose to <em>emphasize</em> some text and <strong>strengthen</strong> other text.</p>
				</div>
			</div>
			<div class="post">
				<div class="post-title">Third Post</div>
				<div class="word-count">Word count: 5</div>
				<div class="excerpt">
					<p>A short post without tags.</p>
				</div>
			</div>
			<div class="post">
				<div class="post-title">WIP</div>
				<div class="word-count">Word count: 7</div>
				<div class="excerpt">
					<p>This one is a work in progress.</p>
				</div>
			</div>
		</main>
	</body>
</html>
